### ISSUE TITLE
Cannot use (or rather do not know how to use) PHPQRCODE

### ISSUE THREAD

**Issue by l0ckm4** [RobThree/TwoFactorAuth#41]
Hi,

I am very new to this project and am struggling to follow the guidelines for implementing phpqrcode.  I believe I have followed the instructions.  Downloaded and copied in phpqrcode.php into the lib folder, created myprovider.php in the QR subfolder of Providers and have the following as my demo script.

```
<?php
        require_once 'loader.php';
        Loader::register('../lib','RobThree\\Auth');

		
		
        use \RobThree\Auth\TwoFactorAuth;
        $mp = new RobThree\Auth\Providers\Qr\MyProvider();
	$tfa = new RobThree\Auth\TwoFactorAuth('MyApp', 6, 30, 'sha1', $mp);
		
        //$tfa = new TwoFactorAuth('MyApp');

        echo '<li>First create a secret and associate it with a user';
        $secret = $tfa->createSecret(160);  
		// Though the default is an 80 bits secret (for backwards compatibility reasons) we recommend creating 160+ bits secrets (see RFC 4226 - Algorithm Requirements)
        echo '<li>Next create a QR code and let the user scan it:<br><img src="' . $tfa->getQRCodeImageAsDataUri('My label', $secret) . '"><br>...or display the secret to the user for manual entry: ' . chunk_split($secret, 4, ' ');
        $code = $tfa->getCode($secret);
        echo '<li>Next, have the user verify the code; at this time the code displayed by a 2FA-app would be: <span style="color:#00c">' . $code . '</span> (but that changes periodically)';
        echo '<li>When the code checks out, 2FA can be / is enabled; store (encrypted?) secret with user and have the user verify a code each time a new session is started.';
        echo '<li>When aforementioned code (' . $code . ') was entered, the result would be: ' . (($tfa->verifyCode($secret, $code) === true) ? '<span style="color:#0c0">OK</span>' : '<span style="color:#c00">FAIL</span>');
?>
```
But I am getting the following error.

`Fatal error: Class 'RobThree\Auth\Providers\Qr\MyProvider' not found in /var/www/html/vendor/robthree/twofactorauth/demo/demo1.php on line 15`

Can anyone help please?

**Comment by RobThree** [comment]
Well, is there a `RobThree\Auth\Providers\Qr\MyProvider.php` file? Did you check the case (files are most likely case-sensitive). What else have you tried?

**Comment by l0ckm4** [comment]
Hi Rob,

I followed the instructions :-)  BUT, it was partly a case thing.  Renamed myprovider.php to MyProvider.php but that gave me an error stating the namespace declaration should be the first entry in the MyProvider.php file.  Moving that up above the require_once line sorted that but then it moaned it could not find the phpqrcode.php file so I had to provide the full path and not a relative one.  After that it worked.  Spent all afternoon on this yesterday as well lol. 

Thanks for the prod in the right direction.
